doc/user/content/get-started.md

```diff
@@ -138,172 +132,307 @@ The first step is to declare where to find and how to connect to the sample Kafk
-     items     | kafka | 3xsmall
-     purchases | kafka | 3xsmall
+            name        |   type    | size
+    --------------------+-----------+------
+     items              | kafka     |
+     items_progress     | subsource |
+     purchases          | kafka     |
+     purchases_progress | subsource |
+    (4 rows)
+    ```
+
+    In addition to `items` and `purchases`, which mirror the Kafka topics named in your `CREATE SOURCE` commands, you'll see `items_progress` and `purchases_progress`. These "subsources" are created by Materialize so you can [monitor source ingestion progress](/sql/create-source/kafka/#monitoring-source-progress). 
+    
+    It's important to note that before an initial snapshot of the data is complete, Materialize won't be able to guarantee correct results. The length of the snapshotting process depends on the size of the source, but for this scenario, the process will complete quickly. 
+
+1. Before moving on, look at a few records from `items` and `purchases` to get a sense of the dataset:
+
+    ```sql
+    SELECT * FROM items LIMIT 1;
     ```
+    <p></p>
+
+    ```nofmt
+      id  |       name       | category  | price  | inventory | inventory_updated_at |      created_at      |     updated_at
+    ------+------------------+-----------+--------+-----------+----------------------+----------------------+---------------------
+     1024 | health spark     | gadgets   | 462.23 |      1365 | 2023-03-06T09:07:22Z | 2023-03-06T09:07:22Z | 2023-03-06 09:07:22
+    (1 rows)
+    ```    
 
-    Creating a source will prompt Materialize to **start ingesting data** from the specified topics into durable storage.
+    ```sql
+    SELECT * FROM purchases LIMIT 1;
+    ```
+    <p></p>
 
-### Transform data
+    ```nofmt
+        id    | user_id | item_id | status | quantity | purchase_price | deleted |      created_at      |     updated_at
+    ----------+---------+---------+--------+----------+----------------+---------+----------------------+---------------------
+     57409536 |    3661 |    1955 |      1 |        5 |         300.65 |       0 | 2023-04-18T22:50:48Z | 2023-04-18 22:50:48
+    (1 row)
+    ```
 
-With data being ingested into Materialize, we can start building SQL statements to transform it into something actionable.
+    Note how the sources can be queried just like tables in a typical SQL database. 
 
-1. Create a [view](/overview/key-concepts/#non-materialized-views) `item_purchases` that calculates rolling aggregations for each item in the inventory:
+<!-- 1. With sources connected, Materialize starts ingesting the Kafka topics and writing their data to durable storage. Before an initial snapshot of the data is complete, Materialize won't be able to guarantee correct results, so let's query Materialize's internal `mz_source_statistics` table to [check the initial snapshotting progress](/ops/troubleshooting/#has-my-source-ingested-its-initial-snapshot) of our sources:
```

Comment:
Why's this commented out? Is it a stray change?

Reply:
Sorry, yes, this was stray. Though now I have a related paragraph elsewhere, commented out because I need help understanding this better.